internal/testutil/acc/cluster.go

```diff
@@ -7,18 +7,29 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 )
 
-func GetClusterInfo(projectID string) (clusterName, clusterNameStr, clusterTerraformStr string) {
+// GetClusterInfo is used to obtain a project and cluster configuration resource.
+// If env variables `MONGODB_ATLAS_CLUSTER_NAME` and `MONGODB_ATLAS_PROJECT_ID` are defined, creation of resources is avoided (useful for local execution)
+func GetClusterInfo(orgID string) (projectIDStr, clusterName, clusterNameStr, clusterTerraformStr string) {
 	// Allows faster test execution in local, don't use in CI
```

Comment:
knitpit: maybe integrate/delete/modify this internal comment with the function comments

Reply:
makes sense, will adjust